ipn/local.go

```diff
@@ -1015,16 +1015,19 @@ func (b *LocalBackend) parseWgStatusLocked(s *wgengine.Status) (ret EngineStatus
 	return ret
 }
 
-// shieldsAreUp returns whether user preferences currently request
-// "shields up" mode, which disallows all inbound connections.
-func (b *LocalBackend) shieldsAreUp() bool {
+// shouldUploadServices returns whether this node should report services
+// in Hostinfo. When the user preferences currently request "shields up"
+// mode, all inbound connections are not allowed, so services are not
```

Comment:
s/are not allowed/are refused/

"so services are not reported" doesn't immediately follow. Maybe "Because all inbound connections are refused, in practice, no services are available, so we do not collect them."

Reply:
Oh yeah great point! I think "are refused" gets this across nicely.